### PR TITLE
AbhiBook

### PR DESCRIPTION
**Pull request by AbhiSivasailam**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -1 +1,167 @@
...
+      <div class="navbar-collapse collapse" id="bs-example-navbar-collapse-1">
+        <div class="col-sm-6 col-sm-offset 3">
+          <input type="text" class="custom-form form-control" placeholder="search for users">
+        </div>
+
+        <div class=" navbar my_name col-sm-offset-10">
+            <p>Harry</p>
+        </div>
+      </div>
+      </nav> <!-- closing nav -->
```
Comment (behaviary) [inline about.html:49]: A personal note: For end of line comments, I find the  easier to write. It is still clear what is going on.

File: about_edit.html [hunk 2]
```diff
@@ -1 +1,178 @@
...
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+
+</head>
+<body>
+<header>
```
Comment (behaviary) [inline about_edit.html:20]: Indent here. This is inside of the body.

File: about_edit.html [hunk 3]
```diff
@@ -1 +1,178 @@
...
+<!-- nav bar -->
+  <div class="container">
+    <div class="row">
+
```
Comment (behaviary) [inline about_edit.html:24]: I don't understand why the nav is inside of a container, inside of a row. I removed them, and it looked the same.

File: about_edit.html [hunk 4]
```diff
@@ -1 +1,178 @@
...
+      <nav class="navbar navbar-fixed-top" role="navigation">
+      <div class="navbar-header">
```
Comment (behaviary) [inline about_edit.html:26]: Indent. ^

File: about_edit.html [hunk 5]
```diff
@@ -1 +1,178 @@
...
+        <div class=" navbar my_name col-sm-offset-10">
+            <p>Harry</p>
+        </div>
+      </div>
+      </nav> <!-- closing nav -->
+    </div><!-- closing row -->
+  </div><!-- closing column -->
+</header>
+<main>
+<!-- nav bar -->
```
Comment (behaviary) [inline about_edit.html:54]: ^confusing that the nav comment is inside of main.

File: about_edit.html [hunk 6]
```diff
@@ -1 +1,178 @@
...
+
+<!-- banner box -->
```
Comment (behaviary) [inline about_edit.html:56]: ha ha I didn't know what you meant by banner box at first. Saw the image and realized.

File: about_edit.html [hunk 7]
```diff
@@ -1 +1,178 @@
...
+          </button>
+        -->
+
+        <ul class="jumbotron-links"> <!-- NOTE: STYLE INLINE-BLOCK CSS-->
+            <li><a href="timeline.html">Timeline</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="photos.html">Photos <span class="counter">(123)</span></a></li>
+            <li><a href="friends.html">Friends <span class="counter">(542)</span></a></li>
+            <li class="pull-right present_tab"><a href="about_edit.html">Edit Profile</a></li>
+        </ul>
```
Comment (behaviary) [inline about_edit.html:87]: I like the list implementation of this over the buttons.

File: about_edit.html [hunk 8]
```diff
@@ -1 +1,178 @@
...
+<div class="container about-page-container">
+  <div class="box-header">
+    <div class="row">
+          <h2 class="col-sm-2 col-sm-offset-5">About</h2>
+          <div class="col-sm-2 col-sm-offset-3">
+            <button class="btn-default pull-right edit-button"><a href="about_edit.html">Edit your profile</a></button>
+          </div>  
+    </div>
+  </div>
+    <div class="col-sm-12 about-content">
```
Comment (behaviary) [inline about_edit.html:108]: unnecessary indent. line 108.

File: about_edit.html [hunk 9]
```diff
@@ -1 +1,178 @@
...
+      <div class="row">
+          <section class="col-sm-5 edit-left-panel">
+
+            <h3>Basic Information</h3>
+
```
Comment (behaviary) [inline about_edit.html:113]: Oh no! Not tables!

File: about_edit.html [hunk 10]
```diff
@@ -1 +1,178 @@
...
+            <td>harry_potter@hogwarts.edu</td>
+            </tr>
+
+            <tr>
+            <td><strong>Telephone </strong></td>
+            <td><input type="text" class="custom-form form-control" placeholder="555-123-4567"></td>
+            </tr>
+            </table>
+          </section>
+
```
Comment (behaviary) [inline about_edit.html:149]: The tables turned out pretty good. I have had nothing but trouble with tables in the past.

File: about_edit.html [hunk 11]
```diff
@@ -1 +1,178 @@
...
+</div><!-- Closing container -->
+
+
+<!-- profile panel -->
+
+</main>
+
+</body>
+
+</html><!-- Your markup here! -->
```
Comment (behaviary) [inline about_edit.html:178]: indents, already mentioned on the other tags.

File: friends.html [hunk 12]
```diff
@@ -1 +1,177 @@
...
+        </div>
+      </div>
+  </div>
+</div>
+
+<!-- banner box -->
+
+<!-- profile panel -->
+
+<div class="container about-page-container">
```
Comment (behaviary) [inline friends.html:99]: .about-page-container, eh? Semantic tags can become confusing when they are reused on other pages. I think I did that on mine too. :8ball:

File: friends.html [hunk 13]
```diff
@@ -1 +1,177 @@
...
+
+
+          <div class="friend-tile second-row col-sm-5">
+             <img src="images/default_photo.png">
+              <p><a href="#">Hermione Granger</a><br>432 Friends</td></p>
+              <button type="button" class="btn">Unfriend Me</button>
+          </div>
+
+        </div>
+
```
Comment (behaviary) [inline friends.html:138]: 107-150 :+1: the code is well organized and it is a good implementation.
Really nice except for one confusing style: all of your <a> tags are styled white! Poor hermione's name is Evanesco.

File: friends.html [hunk 14]
```diff
@@ -1 +1,177 @@
...
+             <img src="images/default_photo.png">
+              <p><a href="#">Hermione Granger</a><br>432 Friends</td></p>
+              <button type="button" class="btn">Unfriend Me</button>
+          </div>
+        </div>
+
+    </div><!-- closing row -->
+      </div><!-- closingcolspace -->
+</div><!-- Closing container -->
+
```
Comment (behaviary) [inline friends.html:155]: Woah there! Crazy divs...^

File: index.html [hunk 15]
```diff
@@ -0,0 +1,260 @@
...
+					<button type="button" class="btn btn-success btn-lg btn-block" id="sign_up_button">Sign Up!</button>
+				</form>
+			</section>
+		</div><!-- / main row-->
+	</main>
+
+
+</body>
+
+</html>
```
Comment (behaviary) [inline index.html:260]: :smile:

File: photos.html [hunk 16]
```diff
@@ -1 +1,180 @@
...
+							</td>
+							<td><figure><img src="images/default_user.gif" class="photo-page-img">
+							<figcaption>Uploaded January 16, 2015</figcaption></figure>
+							</td>
+						</tr>
+						</table>
+      </div><!-- closing row -->
+      </div><!-- closingcolspace -->
+</div><!-- Closing container -->
+
```
Comment (behaviary) [inline photos.html:172]: I'll take it. Harry Potter's face is scaring me on this page.  I would add some width styles to the images on this page so they don't jump out of the div. Also tables :zap: will cause problems.

File: timeline.html [hunk 17]
```diff
@@ -1 +1,323 @@
...
+								<td><img src="images/default_user.gif"></td>
+								<td><img src="images/default_user.gif"></td>
+								<td><img src="images/default_user.gif"></td>
+							</tr>
+							<tr>
+								<td><img src="images/default_user.gif"></td>
+								<td><img src="images/default_user.gif"></td>
+								<td><img src="images/default_user.gif"></td>
+							</tr>
+						</table>
```
Comment (behaviary) [inline timeline.html:144]: You may have a different screen size than I do. This is what I see initially: 
screen shot 2014-09-15 at 10 02 12 pm
This is a scaling issue though.

File: timeline.html [hunk 18]
```diff
@@ -1 +1,323 @@
...
+        </ul>
+
+
+        </div>
+      </div>
+  </div>
+</div>
+
+<!-- banner box -->
+
```
Comment (behaviary) [inline timeline.html:96]: Crazy indents.

File: css/styles.css [hunk 19]
```diff
@@ -0,0 +1,348 @@
...
+
+}
+.jumbotron-links a {
+	color:black;
+}
+.my_name p {
+font-size: .8em;
+margin-top: 10px;
+margin-bottom: -2px;
+}
```
Comment (behaviary) [inline css/styles.css:348]: see convo for css comments :shipit:

**Comment by behaviary** [comment]
Overall great job! :+1: This project was big and difficult and you guys shipped it on time. That gets mad kudos. I left some inline comments for you to view at your leisure. The high level things that need a little love. First,  indenting - I know exactly why this is a problem: Sublime Text. When you copy and paste things in sublime, it goes absolutely wonky. That being said, keep in mind that every time you open a new tag the child should be tabbed in. This includes the <body> tag.

On the CSS:
Did you abandon SaSS? It looks like Styles1 is the compiled CSS file, and you wrote styles.css from scratch. Keep it sassy, San Diego. 

It looks like you are fighting bootstrap quite a bit.  You custom styled the width of 9 classes/ids, which can be done with bootstraps grid system. This is true for the elements that you have added padding to as well (18 classes/IDs): columns have 15px auto padding, and most things larger could be solved with offset. The weird thing in bootstrap is the purpose of .row is almost exclusively to remove that 15px padding from containers and columns.

Your solution works, you shipped it, and it looks pretty good. You get a :cake:+
